torchx/schedulers/aws_batch_scheduler.py

```diff
@@ -252,7 +338,7 @@ class AWSBatchOpts(TypedDict, total=False):
     user: str
     image_repo: Optional[str]
     share_id: Optional[str]
-    priority: Optional[int]
+    priority: int
```

Comment:
why is priority not-optional?

Reply:
it defaults to 0 in _runopts(). Priority isn't actually used if share_id is None